docs/stack/client/querying.md

```diff
@@ -0,0 +1,124 @@
+# Query, Observables & Subscription
+
+In this guide we will describe how to query and subscribe to the subgraph that contains the index of DAOstack data.
+
+## Query
+These are the graphQL queries sent to graphnode to fetch DAOstack data from the subgraph.
+
+Following are the two ways to use client to query the subgraph:
+
+### Standard queries
+These are the queries provided by the *Entity methods*. The entity methods return an Observable for our queries as [described below](#observables-subscription).
+
+        // proposals of the DAO
+        Proposal.search(arc, { where: { dao: "0x123" } })
+
+        // members of the given dao
+        dao.members()
+
+### Custom queries
+  These are explicit queries made using the static method `arc.apolloClient`.
+  These queries can be customized to have more control over what data gets fetched and follow the standard
+  [graphQL syntax](../subgraph/queries/)
+
+        arc.apolloClient.sendQuery(gql`query {
+            proposal (id: "0x1245") {
+              votes: { id outcome }
+            }}`)
+
+  
+## Observables & Subscription
+The entity methods provided by `@daostack/client` for querying the subgraph, by themself does not actually send the query to the server. Instead, each methods returns an *Observable* to which we can subscribe. *Subscriptions* are used so that our handler gets called every time a new value is emitted for the observable stream.
+
+Take a look at the following methods that return observable:
+```
+dao.proposals()
+proposal.state()
+```
+
+Now, in order to query the server we must subscribe
+```
+const observable = arc.daos()
+// only in the next line will a query be sent to the server
+const subscription = arc.daos.subscribe((daos) => console.log(`we found ${daos.length} results`))
+```
+
+There are multiple ways to subscribe to the Observables
+
+  - Subscribe to Cache
+  - Subscribe to Server standard queries
+  - Subscribe to Explicit queries
```

Comment:
"Explicit" should be "explicit"

Distinction between "explicit" "server standard"?  Is explicit also to the "server"?